subscriptions/src/subscription/CachedSubscription.ts

```diff
@@ -11,12 +11,19 @@ export default class CachedSubscription extends Subscription {
         this._cacheKey = cacheKey;
     }
 
+    protected _cachedSubscription = {};
+
+    public async loadCache() {
```

Comment:
I think we can keep getItemSync api in cache, so we don't need to update CacheSubscription to minimize changes.